.circleci/config.yml

```
@@ -3,7 +3,7 @@ version: 2
 jobs:
   lint:
     docker:
-      - image: circleci/node:lts
+      - image: circleci/node:14.17.1
```

What about this? Ref: https://circleci.com/docs/2.0/circleci-images/
```suggestion
      - image: circleci/node:latest
```